web/.eslintrc.json

```diff
@@ -17,6 +17,7 @@
     "react/jsx-filename-extension": 0,
     "react/require-default-props": 0,
     "react/forbid-prop-types": 0,
+    "react/destructuring-assignment": 0,
```

Comment:
I fixed `CheckboxGroup` to use destructuring (`const { children, ...rest } = props;`) but that broke another rule which was `no-unused-vars` (since `...rest` wasn't used).

But now I realize that `...rest` is optional :stuck_out_tongue: , I'll make this change.